pipelines/controller/src/test/java/org/openmrs/analytics/ControlPanelApplicationTests.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
are we going to have tests for the other Java classes?

Reply:
Yes, this is added by default Spring Boot app creator. I should add many other unit-tests for other classes (in future PRs).